docs/content/concepts/authorization/authorizers.md

```diff
@@ -5,46 +5,89 @@ description: >
 
 # Authorizers
 
-The following authorizers are configured in kcp:
+In kcp, a request has four different ways of being admitted:
 
-| Authorizer                             | Description                                                                       |
-|----------------------------------------|-----------------------------------------------------------------------------------|
-| Top-Level organization authorizer      | checks that the user is allowed to access the organization                        |
-| Workspace content authorizer           | determines additional groups a user gets inside of a workspace                    |
-| Maximal permission policy authorizer   | validates the maximal permission policy RBAC policy in the API exporter workspace |
-| Local Policy authorizer                | validates the RBAC policy in the workspace that is accessed                       |
-| Kubernetes Bootstrap Policy authorizer | validates the RBAC Kubernetes standard policy                                     |
+* It can be made to one of the preconfigured paths that do not require authorization, like `/healthz`.
+* It can be performed by a user in one of the configured always-allow groups, by default `system:masters`.
```

Comment:
`system:masters` and `system:kcp:admins`. Where first should never be used by users itself. For external users second one is intended to be used.

Reply:
https://github.com/kcp-dev/kcp/blob/main/pkg/server/options/authorization.go#L48 says it's only 1 default group though.